elasticsearch/Dockerfile.origin

```diff
@@ -1,4 +1,4 @@
-FROM ubi7:latest
+FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-base-openshift-4.6
```

Comment:
:-1: why are we moving our origin source to be something that requires auth?

Reply:
Until ART fixes the general public availability for our base images, we need to comply with this limitation. The tenet is here to continue building locally with UBI8 images and catch any build errors at development time.

Reply:
im not sure i entirely agree with this thought... i can build using the `ubi7:latest` image locally now

Reply:
It is not about what you are able to build now, because it is possible to build with UBI7 and UBI8 today. It is more about using UBI8 as it will constantly get sync'ed from ART into our CI from 4.6 on. Thus, we should work on that version from now on to discover locally if anything breaks with ART updates to UBI8.

Reply:
going back to my original question -- why are we updating our origin image to use a base image that requires authentication?
why not just make this `ubi8:latest` if we're concerned with building with ubi8?

Reply:
IC, now I get your point. You meant `ubi8:latest`, I understood `ubi7:latest`.